scripts/c4dev/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM timbru31/java-node:11-alpine-jdk
```

Comment:
For now I will accept it but in future in spare time I will change this docker image base to something less mutable

Reply:
All right